Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
+MAINTAINER Lukasz Gornicki <derberg@wp.pl>
+# add bash and git
+RUN apk update && apk add --no-cache bash git openssh && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
+# clone and install skeleton
+RUN git clone https://github.com/YaaS/docpad-skeleton-apidocs.git
+WORKDIR /docpad-skeleton-apidocs
+# install dependencies of the skeleton
+RUN npm run prepare
+# initialize your skeleton with sample data
+RUN npm run init
+# volume name
+VOLUME /docpad-skeleton-apidocs
+EXPOSE 9778
```

Comment:
Not completely different approach, at this moment `CMD` is set to `node` - inherited from base image, so if someone run container as mentioned above it will see repl node. Good practice is set `CMD`, it can prevent problems, `Dockerfile` is more self documenting and predictable then.